exec/init_expr.go

```diff
@@ -36,9 +36,10 @@ func execInitExpr(expr []byte, globals []int64) int64 {
 
 	for {
 		b, err := r.ReadByte()
-		if err == io.EOF {
+		switch err {
+		case io.EOF:
 			break
-		} else if err != nil {
+		default:
```

Comment:
`nil` will fall under default and cause a panic

Reply:
Good catch. Thanks.